src/components/Nav/Nav.js

```diff
@@ -2,22 +2,33 @@ import React from 'react';
 import { Link } from "react-router-dom";
 import './Nav.scss';
 
-function Nav({ user }) {
+function Nav({ user, accountType }) {
 
   return <nav>
     <section>
     <Link to="/areas">
       <h1>VRad!</h1>
     </Link>
     <Link to="/favorites">
-      View Favorite Listings
+      View Favorite Listings: ??
     </Link>
     </section>
+    <section>
       <p>Welcome, {user || "friend!"}</p>
+      <p>{accountTypeMessage(accountType) || "This is VRad!"}</p>
+    </section>
     <Link to="/">
       <button>Sign Out</button>
     </Link>
   </nav>
 }
 
+function accountTypeMessage(accountType) {
```

Comment:
This function shouldn't live here - we should either be performing the function before the nav is rendered and just passing in the appropriate string, or this function should get passed in here as a prop and called with the appropriate params.

Reply:
This is where I was questioning if we need a class component since there are helper functions

Reply:
So, this where we have to decide how we are going to determine the user type message. Do you want this method to live in the App component?

Reply:
I think that making a class component just to pass one function is overkill, especially when this component doesn't need state. I honestly don't know if you even need a function at all - you could just pass an interpolated string that said something like "Welcome, ${user}. Your account type: ${accountType}.

Reply:
But if you did want the method, then yes I think having the function live on App and get passed as a prop is fine.

Reply:
I'll make the change to: Your account type: ${accountType}

Reply:
Changed the display message of account type